src/pages/QuestPage/QuestPage.jsx

```diff
@@ -126,43 +122,51 @@ const QuestPage = () => {
       setEligibiltyStatusBeforeCheck(false);
       setWalletAddress(address);
       const addressLastChar = getLastCharacterOfAString(address);
-      dispatch(fetchNFTContestData({ addressLastChar, chainId })).then(
-        (res) => {
-          const nftData = res?.payload?.data?.find(
-            (resData) => resData.wallet_address === address,
-          );
-          if (nftData && Object.keys(nftData).length) {
-            setStateForAccountDetailsForNFT(nftData);
-            dispatch(setUserEligibleNFTAction(true));
+      dispatch(fetchNFTContestData({ addressLastChar, chainId })).then(res => {
+        const nftData = res?.payload?.data?.find(
+          resData => resData.wallet_address === address
+        );
+        if (nftData && Object.keys(nftData).length) {
+          setStateForAccountDetailsForNFT(nftData);
+          dispatch(setUserEligibleNFTAction(true));
+          if (isFakeContract) {
+            const configResponse = config(chainId);
+            setMintData(configResponse.data);
+          } else {
             setMintData({
-              token_id: nftData?.calldata[0],
-              proof: nftData?.proof,
+              token_id: 5,
```

Comment:
@vnaysngh-mudrex If the fake contract is not true then it will always set mintdata as a hardcoded one, Doesn't it take it from missions list data?

Reply:
I don't understand the question, sorry. Can you elaborate pls